src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -158,4 +159,40 @@ Scalar<DataVector> area_element(
     const tnsr::i<DataVector, 3, Frame>& normal_one_form,
     const DataVector& radius,
     const tnsr::i<DataVector, 3, Frame>& r_hat) noexcept;
+
+/*!
+ * \ingroup SurfacesGroup
+ * \brief Spin function of a 2D `Strahlkorper`.
+ *
+ * \details See Eqs. (2) and (10)
+ * of https://arxiv.org/abs/1708.07325.
```

Comment:
Will SpinFunction be used in determining the spin axis in SpECTRE? I would include a comment mentioning what "SpinFunction" will be useful for/ most used to do - if it's meant to implement a method from this paper, or something else

Reply:
I hope my comments now clarify what the spin function is and why it is useful.